### PR TITLE
#353: Remove Comparable<Text> into Text, new class ComparableText

### PR DESCRIPTION
**Pull request by Zefick**
I noticed that the task was stuck somewhere in August and `Text` still implements `Comparable<Text>`. That's my contribution.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this pull request

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#496` assigned to @kkamkou  ([profile](http://www.0crat.com/u/kkamkou})). The budget is [fixed](http://datum.zerocracy.com/pages/policy.html#4) and it is 30 minutes. Please, read the [Policy](http://datum.zerocracy.com/pages/policy.html) and go ahead.

**Comment by 0crat** [comment]
@kkamkou this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [par.8](http://datum.zerocracy.com/pages/policy.html#8).

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#496` assigned to [@llorllale](http://www.0crat.com/u/llorllale}). The budget is [fixed](http://datum.zerocracy.com/pages/policy.html#4) and it is 30 minutes. Please, read [§4](http://datum.zerocracy.com/pages/policy.html#4), [§8](http://datum.zerocracy.com/pages/policy.html#8), and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

File: src/main/java/org/cactoos/text/ComparableText.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * )
+ * </pre>
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Sergey Sharov (zefick@mail.ru)
+ * @version $Id$
+ * @since 0.27
+ */
+public final class ComparableText implements Text, Comparable<ComparableText> {
```
Comment (llorllale) [inline src/main/java/org/cactoos/text/ComparableText.java]: `ComparableText` should implement `Comparable<Text>`

**Comment by llorllale** [comment]
@yegor256 I approve this PR

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
The job [#496](https://github.com/yegor256/cactoos/issues/496) is now out of scope

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to [@llorllale](http://www.0crat.com/u/llorllale), total is [+45](http://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to [@yegor256](http://www.0crat.com/u/yegor256), total is [+6990](http://www.0crat.com/u/yegor256)

**Pull request merged**